<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Saifuls-MacBook-Pro.local" failures="1" tests="4" name="Test" time="0.056" errors="0" timestamp="2022-10-06T23:22:31 EDT">
  <testcase classname="testNG.SecondTestCases" name="setup" time="0.009">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testNG.SecondTestCases.setup(SecondTestCases.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- setup -->
  <testcase classname="testNG.SecondTestCases" name="searchcustomer" time="0.0">
    <skipped/>
  </testcase> <!-- searchcustomer -->
  <testcase classname="testNG.SecondTestCases" name="addcustomer" time="0.0">
    <skipped/>
  </testcase> <!-- addcustomer -->
  <testcase classname="testNG.SecondTestCases" name="tearDown" time="0.002"/>
</testsuite> <!-- Test -->
